src/subset.c

```diff
@@ -77,7 +77,6 @@ static void subsetVectorRaw(SEXP ans, SEXP source, SEXP idx, const bool anyNA)
   case CPLXSXP : {
     Rcomplex *sp = COMPLEX(source);
     Rcomplex *ap = COMPLEX(ans);
-    Rcomplex NA_CPLX = { NA_REAL, NA_REAL };
```

Comment:
looking at it again, would just inlining `= (Rcomplex){NA_REAL, NA_REAL}` do the trick?

Reply:
I don't think `(Rcomplex){NA_REAL, NA_REAL}` will work. Because `Rcomplex` is a `struct`. The {NA_REAL, NA_REAL} is special syntax just for initialization of a variable/struct.